Makefile

```diff
@@ -351,20 +356,20 @@ deploy-minio:
 
 # generic end-to-tests
 .PHONY: prepare-e2e
-prepare-e2e: kuttl start-kind cert-manager set-test-image-vars build docker-build load-image-operator deploy
+prepare-e2e: chainsaw start-kind cert-manager set-test-image-vars build docker-build load-image-operator deploy
 
 .PHONY: e2e
-e2e:
-	$(KUTTL) test
+e2e: prepare-e2e
```

Comment:
Not sure why there was no dependency between targets here 🤔

Reply:
This is because prepare-e2e only prepares an e2e setup on `kind`, but e.g. when I want to run e2e tests on OpenShift or minikube, I do not want to run the prepare-e2e task.

Reply:
Hmm, but in the workflow we run `make prepare-e2e e2e KUBE_VERSION=$KUBE_VERSION`.
How is targets ordering working in this case ?

Reply:
It worked without issues, I think if there are no complex dependencies defined it just executes the targets in the order they are defined. I guess.

Reply:
`make` will try to run targets concurrently if there's no dependency. I will remove the dependencies i added 🤷

Reply:
But priority is to get green results for now 😂

Reply:
Should be fixed now 🤞

Reply:
Yeah, green 🎉

Reply:
afaics the default in make is serial execution unless `-j` is specified.
Fixing the makefile to support parallel execution is a task for another day :D

Reply:
Agree 💯 i reverted my changes to the makefile and everything is passing now.
I will look deeper in follow-up PRs to make use of chainsaw features in the tests and remove scripts when possible.